conan/api/subapi/workspace.py

```diff
@@ -201,6 +203,45 @@ def add(self, path, name=None, version=None, user=None, channel=None, cwd=None,
         self._ws.add(ref, full_path, output_folder, product)
         return ref
 
+    @staticmethod
+    def initialize(path):
+        abs_path = make_abs_path(path)
+        os.makedirs(abs_path, exist_ok=True)
+        ws_yml_file = Path(abs_path, WORKSPACE_YML)
+        ws_py_file = Path(abs_path, WORKSPACE_PY)
+        if not ws_yml_file.exists():
+            ConanOutput().success(f"Created empty {WORKSPACE_YML} in {path}")
+            save(ws_yml_file, "")
+        if not ws_py_file.exists():
+            ConanOutput().success(f"Created basic {WORKSPACE_PY} in {path}")
+            save(ws_py_file, textwrap.dedent('''\
```

Comment:
This looks closer to a ``conan new`` than to an ``conan workspace init``, it is a full template, shouldn't it be more minimalistic?